### PR TITLE
Feature/deploy vhd

### PR DESCRIPTION
**Pull request by neolynx**

### CODE REVIEW

File: plugins/vhd.plugin [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+init_deployment_vhd()
+{
+  vmname=$(get_deployment_filename "vhd")
+  deployment=$vmname
+
+  # FIXME check VMSIZE
+  echo " * creating VHD disk of ${VMSIZE}M"
+  VMDISK=$WORK_DIR/$deployment
+  qemu-img create -f vpc -o subformat=fixed,force_size $VMDISK ${VMSIZE}M >&2
```
Comment (szakalboss) [inline plugins/vhd.plugin:19]: any default VMSIZE?

File: plugins/vhd.plugin [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  if [ $? -ne 0 ]; then
+    log_error "Error creating $VMDISK"
+  fi
```
Comment (hus0) [inline plugins/vhd.plugin:22]: Should we use `exit_on_error` instead?

**Review by hus0 — CHANGES_REQUESTED** [review]

File: plugins/vhd.plugin (hunk 2 above)
Comment (neolynx) [reply plugins/vhd.plugin:22]: yes, but will make a separate PR, for changing everything :)

Comment (hus0) [reply plugins/vhd.plugin:22]: Okay, thanks.

**Review by hus0 — APPROVED** [review]

**Review by szakalboss — APPROVED** [review]

**Review by szakalboss — APPROVED** [review]

**Pull request merged**